src/Validator/Validators/hdtXSDValidator.cpp

```
@@ -303,20 +304,29 @@ namespace hdt
+		PhysicsXmlSource src = readAndExpandPhysicsXml(xmlPath);
 
-		if (bytes.empty()) {
+		if (src.xml.empty()) {
 			result.isValid = false;
 			result.violations.push_back({ xmlPath, 0, 0, "", "File not found or empty" });
 			return result;
 		}
+		if (!src.ok) {
+			for (const auto& d : src.diags)
+				result.violations.push_back({ xmlPath, d.line, 0, "", "pattern expansion: " + d.message });
+			if (src.diags.empty())
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)